### ISSUE TITLE
Fix a crash for https://localhost:44374/Genesis.4_1/ACRAEM

### ISSUE THREAD

**Issue by pmachapman** [pmachapman/GoTo.Bible#19]
```
fail: GoToBible.Web.Server.Controllers.RenderPassageController[0]
URL: http://goto.bible/RenderPassage
Referer: https://goto.bible/Acts.9_1/ACRAEM/ENGEVD
RenderingParameters: {"BackgroundColour":{"R":0,"G":0,"B":0,"A":0,"IsKnownColor":false,"IsEmpty":true,"IsNamedColor":false,"IsSystemColor":false,"Name":"0"},"IsDebug":false,"Font":{"Bold":false,"FamilyName":"Calibri","Italic":false,"SizeInPoints":14.25,"Strikeout":false,"Underline":false},"ForegroundColour":{"R":0,"G":0,"B":0,"A":0,"IsKnownColor":false,"IsEmpty":true,"IsNamedColor":false,"IsSystemColor":false,"Name":"0"},"Format":1,"InterlinearIgnoresCase":true,"InterlinearIgnoresDiacritics":true,"InterlinearIgnoresPunctuation":true,"PassageReference":{"End":"","IsValid":true,"Original":"Acts 9:1","Start":"Acts 9:1"},"PrimaryProvider":"DigitalBiblePlatformApi","PrimaryTranslation":"ACRAEM","RenderItalics":true,"SecondaryProvider":"DigitalBiblePlatformApi","SecondaryTranslation":"ENGEVD"}
System.InvalidOperationException: Sequence contains no matching element
at System.Linq.ThrowHelper.ThrowNoMatchException()
at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
at GoToBible.Providers.DigitalBiblePlatformApi.GetChapterAsync(String translation, String book, Int32 chapterNumber) in D:\source\repos\GoTo.Bible\GoToBible.Providers\DigitalBiblePlatformApi.cs:line 263
at GoToBible.Engine.Renderer.RenderAsync(RenderingParameters parameters, Boolean renderCompleteHtmlPage) in D:\source\repos\GoTo.Bible\GoToBible.Engine\Renderer.cs:line 106
at GoToBible.Web.Server.Controllers.RenderPassageController.Post(RenderingParameters parameters, Boolean renderCompleteHtmlPage) in D:\source\repos\GoTo.Bible\GoToBible.Web\Server\Controllers\RenderPassageController.cs:line 103

```

**Comment by pmachapman** [comment]
```
fail: GoToBible.Web.Server.Controllers.RenderPassageController[0]
URL: http://goto.bible/RenderPassage
Referer: https://goto.bible/Acts.9_1/ACRAEM/ENGEVD
RenderingParameters: {"BackgroundColour":{"R":0,"G":0,"B":0,"A":0,"IsKnownColor":false,"IsEmpty":true,"IsNamedColor":false,"IsSystemColor":false,"Name":"0"},"IsDebug":false,"Font":{"Bold":false,"FamilyName":"Calibri","Italic":false,"SizeInPoints":14.25,"Strikeout":false,"Underline":false},"ForegroundColour":{"R":0,"G":0,"B":0,"A":0,"IsKnownColor":false,"IsEmpty":true,"IsNamedColor":false,"IsSystemColor":false,"Name":"0"},"Format":1,"InterlinearIgnoresCase":true,"InterlinearIgnoresDiacritics":true,"InterlinearIgnoresPunctuation":true,"PassageReference":{"End":"","IsValid":true,"Original":"Acts 9:1","Start":"Acts 9:1"},"PrimaryProvider":"DigitalBiblePlatformApi","PrimaryTranslation":"ACRAEM","RenderItalics":true,"SecondaryProvider":"DigitalBiblePlatformApi","SecondaryTranslation":"ENGEVD"}
System.InvalidOperationException: Sequence contains no matching element
at System.Linq.ThrowHelper.ThrowNoMatchException()
at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
at GoToBible.Providers.DigitalBiblePlatformApi.GetChapterAsync(String translation, String book, Int32 chapterNumber) in D:\source\repos\GoTo.Bible\GoToBible.Providers\DigitalBiblePlatformApi.cs:line 263
at GoToBible.Engine.Renderer.RenderAsync(RenderingParameters parameters, Boolean renderCompleteHtmlPage) in D:\source\repos\GoTo.Bible\GoToBible.Engine\Renderer.cs:line 106
at GoToBible.Web.Server.Controllers.RenderPassageController.Post(RenderingParameters parameters, Boolean renderCompleteHtmlPage) in D:\source\repos\GoTo.Bible\GoToBible.Web\Server\Controllers\RenderPassageController.cs:line 103

```

**Comment by pmachapman** [comment]
Fixed in e35502465b5dbe36c25cb1c562b0ea35d0ad2126
